src/persist-client/src/write.rs

```diff
@@ -454,9 +454,16 @@ where
         let desc = Description::new(lower, upper, since);
 
         let (mut parts, mut num_updates) = (Vec::new(), 0);
+        let mut runs = vec![];
```

Comment:
supernit: fold this into the above line

Reply:
Do you prefer that declaration style? I always found it a bit less direct to have the names separated from the values like that.

Anyways probably worth it here just for consistency...

Reply:
no strong preference on the style, just a vote for consistency